invokeai/frontend/web/src/features/gallery/components/ImageGrid/GalleryImage.tsx

```diff
@@ -144,6 +148,20 @@ const GalleryImage = (props: HoverableImageProps) => {
           onMouseOver={handleMouseOver}
           onMouseOut={handleMouseOut}
         >
+          <Text
```

Comment:
You'll need to move this `Text` component inside the fragment (the `<>...</>`) so that `IAIDndImage` has a single child